### ISSUE TITLE
Eyes can not  move ?

### ISSUE THREAD

**Issue by jokerlc** [Hangz-nju-cuhk/Talking-Face_PC-AVS#14]
hi, thanks for your great job ! when I test with my own video,  i found that the eyes in the generated video didn't move with the driven video. Is it normal ?

**Comment by Hangz-nju-cuhk** [comment]
Yes, it is. We do not model the eyes thus this is a problem to be tackled.
